docs/design/authentication-patterns.md

```diff
@@ -0,0 +1,68 @@
+# Authentication
+
+Use  dialogs to present authentication screens to your end users. Our flows provide a seamless experience that integrates the Microsoft sign-in with your brand.
+
+## Best practices
+
+<br/>
+
+|Do          |Don't       |
+|:-----------|:-----------|
+| Utilize the space to reinforce your company's brand. Azure Active Directory allows customization of sign-in and access panel pages for consistent look and feel with your services. [Learn More](https://azure.microsoft.com/en-us/documentation/articles/active-directory-add-company-branding/).| Don't overcrowd the task pane or dialogs with buttons.|
+Target your dialogs to a few key widths or breakpoints for better responsive design. [Learn more](https://msdn.microsoft.com/windows/uwp/layout/screen-sizes-and-breakpoints-for-responsive-design)|                      
+
+<br/>
+
+Apply the following patterns as applicable to create or enhance the authentication experience for your add-in.
+
+<br/>
+
+## Authentication flow for a single or multi- identity provider(s)
+
+If SSO is not available to a user, an alternative authentication flow is for a single or multi-identity provider.
+
+<br/>
+
+*1. First Run Placemat - Place the branded sign-in button as a clear call-to action inside your add-in's UI.*
+
+![Authentication Flow - First run placemat](../images/add-in-FRE-value.png)
+
+<br/>
+
+*2. Provider Sign-in - The identity provider will have their own UI. Microsoft Azure Active Directory allows customization of sign-in and access panel pages for consistent look and feel with your service.* [Learn More](https://azure.microsoft.com/en-us/documentation/articles/active-directory-add-company-branding/)
+
+![Authentication Dialog Single Identity - Provider Sign-in](../images/add-in-multi-authentication-modal.png)
+
+<br/>
+
+*3. Progress - Indicate progress while settings and UI load.*
+
+![Authentication Dialog Single Identity - Progress](../images/add-in-multi-authentication-modal-interstitial.png)
+
+<br>
+
+**Microsoft branded sign-in button**
+
+When using Microsoft's Identity service you'll have the opportunity to use a branded sign-in button that is customizable to light and dark themes. [Learn more](https://azure.microsoft.com/en-us/documentation/articles/active-directory-branding-guidelines/#visual-guidance-for-sign-in).
+
+<br>
+
+## Authentication flow for Single Sign On (SSO)
+
+
+While Single Sign On (SSO) is still in preview, once released in general availability it should be considered for the default authentication flow of your add-in.  SSO provides the user with the best authentication experience, essentially moving authentication into the broader installation process.
+
+As an add-in is being installed, a user will see a consent window similar to the one below:
```

Comment:
I'm not sure this is correct. The consent will either be done in the in client store dialog or by the admin for centralized deployment. Need to confirm with the UX/Identity crew.